.github/workflows/tools_tests.yaml

```diff
@@ -76,7 +76,7 @@ jobs:
         ${{ env.RUN }} "export MAPBOX_TOKEN='pk.eyJ1Ijoiam5ld2IiLCJhIjoiY2xxNW8zZXprMGw1ZzJwbzZneHd2NHljbSJ9.gV7VPRfbXFetD-1OVF0XZg' && \
                         source selfdrive/test/setup_xvfb.sh && \
                         source selfdrive/test/setup_vsound.sh && \
-                        CI=1 pytest tools/sim/tests/test_metadrive_bridge.py -W ignore::pyopencl.CompilerWarning"
+                        CI=1 pytest tools/sim/tests/test_metadrive_bridge.py --time_done=90 -W ignore::pyopencl.CompilerWarning"
```

Comment:
needed to add "-W ignore::pyopencl.CompilerWarning" when running locally as well. let's put this directly in the code so it doesn't need to be specified on the command line (or fix the warning)